malaria-app-android/src/main/java/com/peacecorps/malaria/activities/UserMedicineSettingsFragmentActivity.java

```diff
@@ -103,45 +105,6 @@ public void onCreate(Bundle savedInstanceState) {
 
     }
 
-    @Override
```

Comment:
removed this section because it was the main cause of crash and orientation change problem.